src/Symfony/Component/DependencyInjection/Tests/Compiler/LegacyResolveParameterPlaceHoldersPassTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Tests\Compiler;
+
+class LegacyResolveParameterPlaceHoldersPassTest extends ResolveParameterPlaceHoldersPassTest
```

Comment:
This setup is wrong: you are running all the ResolveParameterPlaceHoldersPassTest twice now: once for ResolveParameterPlaceHoldersPassTest and once for LegacyResolveParameterPlaceHoldersPassTest. LegacyResolveParameterPlaceHoldersPassTest should run only the legacy tests